public/diagnostic.js

```diff
@@ -23,3 +23,36 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Cart = Ember.Object.extend({
+  listOfOrders: [],
+  addToCart : Ember.computed('orders', function() {
+    let orders = Order.get('orders');
+    listOfOrders.pushObject(orders);
+  })
+  totalPrice : Ember.computed('orders.@each.price', function(){
```

Comment:
`totalPrice` should be updated any time price _or_ quantity change; therefore, you either want to watch both sets of properties, or you want to watch the `orderPrice` computed property.